### ISSUE TITLE
Demo mode

### ISSUE THREAD

**Issue by jaegeral** [google/timesketch#2062]
- have a config variable if a server is in demo mode
- demo/demo user will create a dedicated user for you
- will pre populate a jsonl / csv/ plaso as a new sketch

**Comment by joachimmetz** [comment]
maybe instead of building this in as a feature why not have a demo Docker image?

**Comment by jaegeral** [comment]
Yeah we talked about this, the main reason for having a code path is that we want to run this on demo.timesketch.org where we need a special handling for demo user login. Behind the scenes, every demo / demo login will create a new user which is allowed.

Alternative we could indeed run a demo docker image that would create a set (999?) demo users and people would just randomly choose one, but you then might see starred events or commends from someone else then yourself.

Another area that the demo mode needs is the ability to set an ACL flag that a user with that flag (e.g. demo / demo) can add certain things to a sketch (comments, stars) but not upload new data to a sketch.
